pkg/apis/extensions/zz_generated.deepcopy.go

```diff
@@ -43,6 +43,10 @@ func RegisterDeepCopies(scheme *runtime.Scheme) error {
 			return nil
 		}, InType: reflect.TypeOf(&APIVersion{})},
 		conversion.GeneratedDeepCopyFunc{Fn: func(in interface{}, out interface{}, c *conversion.Cloner) error {
+			in.(*AllowedFlexVolume).DeepCopyInto(out.(*AllowedFlexVolume))
```

Comment:
move generated changes to the `generated` commit

Reply:
@wanghaoran1988 Still here.